pathAliases.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
we should probably consume this export in our `jest.config.js` as well.

Reply:
I think it's not required for Jest because Jest consumes -> babelrc config, and babelrc config consumes -> this file. At least I couldn't find a suitable place in `jest.config.js` for this.

Reply:
Oh dang! I'm dumb as hell. You're good to go.